.golangci.yaml

```
@@ -326,4 +326,4 @@ issues:
         - perfsprint
     # TODO: remove after PR is released https://github.com/golangci/golangci-lint/pull/4386
     - text: "fmt.Sprintf can be replaced with string addition"
-      linters: [ perfsprint ]
\ No newline at end of file
+      linters: [ perfsprint ]
```

yamllint

[error] too many spaces inside brackets

(brackets)